tests/Resources/Page/MarketplaceConsole/MarketplaceConsole.robot

```diff
@@ -0,0 +1,67 @@
+*** Settings ***
+Library         SeleniumLibrary
+Library         Collections
+
+*** Keywords ***
+Is SSO Login Page Visible
+  ${login_prompt_visible} =  Run Keyword and Return Status  Page Should Contain    Enter your email to log in
+  [Return]  ${login_prompt_visible}
+
+Get the OC commands For Cluster registration
+    Launch Cluster Tab from Marketplace
+    sleep    5
+    Wait until Element is Visible          //span[contains(text(),'Generate Secret')]        timeout=20
+    Click Button    Generate Secret
+    Wait Until Keyword Succeeds    30  1   Element Should Be Visible     //button[contains(@class, 'disabled')]/span[contains(text(),'Generate Secret')]
+    sleep  5
+    ${elem} =   Get WebElements      xpath://div[@role='textbox']//pre
+    &{List}   	Create Dictionary
+    @{itemname}   Create List     Create namespace         Red Hat Marketplace Subscription          Red Hat Marketplace Kubernetes Secret    Red Hat Marketplace global pull secret
+    FOR  ${idx}  ${ext_link}  IN ENUMERATE  @{elem}   start=0
+
+        Set To Dictionary       ${List}      ${itemname[${idx}]}    ${ext_link.text}
+
+    END
+    [Return]    ${List}
+
+#Launch Marketplace
+#   Open Browser  ${MARKETPLACE_TEST.URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+
+Wait For Marketplace Splash Page
+   Wait Until Page Contains Element    xpath://a[contains(text(), 'Workspace')]   timeout=15
```

Comment:
by Splash page, do you mean the loading page/icon that appears before the actual page? If yes, I think the keyword should look for the specific symbol of loading (see the `Wait for JupyterLab Splash Screen ` keyword for example)

`Wait for JupyterLab Splash Screen
    [Arguments]    ${timeout}=10s    ${sleep}=2s
    [Documentation]    Wait for the JupyterLab splash animation, waiting ``timeout``
    ...    for the splash screen to appear/disappear, then ``sleep``.
    Wait Until Page Contains Element    css:#${JLAB ID SPLASH}    timeout=${timeout}
    Wait Until Page Does Not Contain Element    css:#${JLAB ID SPLASH}    timeout=${timeout}
    Sleep    ${sleep}`

Reply:
I have changed the name to load. Splash give wrong information here. Thanks :)